internal/config/profile.go

```diff
@@ -227,19 +230,52 @@ func (p *profile) SetOrgID(v string) {
 
 // GetConfigDescription returns a map describing the configuration
 func GetConfigDescription(name string) map[string]string {
-	redactedValue := "redacted"
+	settings := viper.GetStringMapString(name)
+	newSettings := make(map[string]string)
```

Comment:
this was because the settings were being edited in place, so it would actually change & potentially save the api key as "redacted" 😰

Reply:
😱